<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000000</width>
    <height>10000000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextBrowser" name="resBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>861</width>
     <height>191</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="fileSelBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_bd">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>11</y>
     <width>221</width>
     <height>140</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="title">
    <string>加密算法</string>
   </property>
   <widget class="QCheckBox" name="cb_Md4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Md4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_Sha1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Sha1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_Sha3_384">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Sha3_384</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_Sha384">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Sha384</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_Md5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Md5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_Sha224">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Sha224</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_Sha256">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Sha256</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_Sha3_512">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Sha3_512</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_Sha512">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Sha512</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_Sha3_224">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Sha3_224</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_All">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_Sha3_256">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>Sha3_256</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_progress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 94);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>521</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="title">
    <string>加密对象</string>
   </property>
   <widget class="QRadioButton" name="rb_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>57</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>纯文本</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>27</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>文件</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="filenameBrowser">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>calBtn</tabstop>
  <tabstop>fileSelBtn</tabstop>
  <tabstop>rb_file</tabstop>
  <tabstop>rb_text</tabstop>
  <tabstop>resBrowser</tabstop>
  <tabstop>cb_All</tabstop>
  <tabstop>cb_Md5</tabstop>
  <tabstop>cb_Md4</tabstop>
  <tabstop>cb_Sha1</tabstop>
  <tabstop>cb_Sha224</tabstop>
  <tabstop>cb_Sha256</tabstop>
  <tabstop>cb_Sha384</tabstop>
  <tabstop>cb_Sha512</tabstop>
  <tabstop>cb_Sha3_224</tabstop>
  <tabstop>cb_Sha3_256</tabstop>
  <tabstop>cb_Sha3_384</tabstop>
  <tabstop>cb_Sha3_512</tabstop>
  <tabstop>lineEdit_bd</tabstop>
  <tabstop>filenameBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
